com.github.rajsolai.response.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.rajsolai.response
+
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+base: io.elementary.BaseApp
+base-version: 'juno-20.08'
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.response
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=x11'
```

Comment:
also if you have `wayland`, you shouldn't have `x11` in most cases. (`fallback-x11` is here as a fallback)

Reply:
Thanks , On it !